### DISCUSSION TITLE
Gambar tidak terload

### DISCUSSION THREAD

**Discussion by DiekiRianMustapa** [teamalgoritma/community#197] [Q&A] [answered]
![Masalah 1](https://user-images.githubusercontent.com/107328516/178165403-6c4eabd3-7205-4370-828a-c090168c28f2.png)

Kenapa gambarnya tidak terload yah?

**Comment by Litaa** [comment, marked as answer]
Halo Bapak Dieki,

Gambar tidak dapat ditampilkan bisa jadi dikarenakan kode program pada file `index.html` yang terdapat pada folder template belum dilengkapi dengan baik.
Kami sangat menyarankan untuk Bapak melakukan pengecekan kembali terkait kelengkapan kode program tsb. 
Terima kasih

**Reply by DiekiRianMustapa** [reply]
![image](https://user-images.githubusercontent.com/107328516/178184455-0f9345eb-ffd5-4f9a-b151-f637691166af.png)
ini sudah saya ikuti sesuai step nya

**Reply by DiekiRianMustapa** [reply]
tapi masih belum ke load gambarnya

**Reply by triarm** [reply]
Hi @DiekiRianMustapa ,
terkait kendala tersebut silahkan Anda untuk memastikan:
1. File index.html sudah disimpan kembali setelah pengeditan
2. Apakah file app.py sudah menghasilkan result3 dan result4?

Demikian yang dapat kami sampaikan. Semoga membantu :)
